packages/stack/src/views/Stack/Card.tsx

```diff
@@ -171,15 +185,18 @@ export class Card extends React.Component<Props> {
 
   private lastToValue: number | undefined;
 
+  private didAnimateOnPush = false;
```

Comment:
here, we need to handle the logic if the route was first preloaded and then navigated. Then, the animation should happen later.